src/technologies/r.json

```
@@ -633,9 +633,9 @@
     "icon": "React Router.svg",
     "implies": "React",
     "oss": true,
-    "scriptSrc": [
-      "(?:/react-router(@|/)([\\d.]+)(?:/[a-z]+)?)?/react-router(?:\\.min)?\\.js\\;version:\\2"
-    ],
+    "js": {
+      "__reactRouterVersion": ""
```

Note: we can distinguish the version with this field, however I'm not sure there's anything Wappalyzer can do with this information

---

Yes we grab the version number.

```suggestion
      "__reactRouterVersion": "([\\d\\.]+)\\;version:\\1"
```